presto-mysql/src/test/java/com/facebook/presto/plugin/mysql/TestMySqlDistributedQueries.java

```diff
@@ -83,6 +83,18 @@ public void testDropTable()
     }
 
     @Test
+    public void testInsertIntoTable()
+            throws Exception
+    {
+        assertUpdate("CREATE TABLE test_insert AS SELECT 123 x, 'insert' y limit 0", 1);
```

Comment:
We can call `execute()` to create the table directly in MySQL (this way is a bit convoluted and harder to follow).